docs/commandline/pouch_rename.md

```diff
@@ -0,0 +1,45 @@
+## pouch rename
+
+Rename a container with newName
+
+### Synopsis
+
+Rename a container object in Pouchd. You can change the name of one container identified by it's name or ID. The container you renamed is ready to be used by it's new name.
```

Comment:
it's ->its

Reply:
That is great. I think we need to merge this and update the source code. When reviewing this pr ran into something wrong, this PR could be used to track issue, then we can arrange resource to fix these issue. 

This part is introduced by you. Could you please submit another PR to fix this? @zeppp